src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/publicApi/model/licence/additionalConditions/AdditionalCondition.kt

```diff
@@ -0,0 +1,50 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.resource.publicApi.model.licence.additionalConditions
+
+import com.fasterxml.jackson.annotation.JsonTypeInfo
+import io.swagger.v3.oas.annotations.media.DiscriminatorMapping
+import io.swagger.v3.oas.annotations.media.Schema
+
+object ConditionTypes {
+  const val ELECTRONIC_MONITORING = "ELECTRONIC_MONITORING"
+  const val MULTIPLE_EXCLUSION_ZONE = "MULTIPLE_EXCLUSION_ZONE"
+  const val STANDARD = "STANDARD"
+}
+
+@Schema(
+  description = "Describes each additional condition on the licence, A discriminator exists to allow specific types of conditions to contain additional info",
+  discriminatorProperty = "type",
+  discriminatorMapping = [
+    DiscriminatorMapping(value = ConditionTypes.STANDARD, schema = StandardAdditionalCondition::class),
+    DiscriminatorMapping(
+      value = ConditionTypes.ELECTRONIC_MONITORING,
+      schema = ElectronicMonitoringAdditionalCondition::class,
+    ),
+    DiscriminatorMapping(
+      value = ConditionTypes.MULTIPLE_EXCLUSION_ZONE,
+      schema = MultipleExclusionZoneAdditionalCondition::class,
+    ),
+  ],
+)
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
```

Comment:
How does `@JsonTypeInfo` being declared on the interface here relate to the consequent declarations for each of the additional conditions type (e.g. `@JsonTypeName(ConditionTypes.ELECTRONIC_MONITORING)` for the `ElectronicMonitoringAdditionalCondition`)